index.html

```diff
@@ -0,0 +1,162 @@
+<!DOCTYPE html>
+<html lang="en">
+
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="icon" href="src/assets/icons/tk.jpg">
+        <title>CV Tukach Kirill</title>
+        <link rel="stylesheet" href="style.scss">
+    </head>
+
+    <body>
+        <header class="header">
+            <a href="#" class="logo">logo</a>
+            <div class="container header-container">
+                <nav class="nav">
+                    <ul class="nav-list">
+                        <li class="nav-item">
+                            <a href="#my-contacts" class="nav-link">My contacts</a>
+                        </li>
+                        <li class="nav-item">
+                            <a href="#about-me" class="nav-link">About me</a>
+                        </li>
+                        <li class="nav-item">
+                            <a href="#my-skills" class="nav-link">My skills</a>
+                        </li>
+                        <li class="nav-item">
+                            <a href="#education" class="nav-link">Education</a>
+                        </li>
+                        <li class="nav-item">
+                            <a href="#experience" class="nav-link">Experience</a>
+                        </li>
+                        <li class="nav-item">
+                            <a href="#languages" class="nav-link">languages</a>
+                        </li>
+                    </ul>
+                </nav>
+            </div>
+        </header>
+        <main class="main">
+            <div class="container main-container">
+
+                <section class="section-left section-profile">
+                    <article class="section-profile_avatar">
+                        <img src="src/assets/photos/photo.jpg" alt="avatar" width="200" class="profile-avatar">
+                    </article>
+                    <article class="section-profile_title">
+                        <h1 class="profile-name">Tukach Kirill</h1>
```

Comment:
You have warning  _ Consider using the h1 element as a top-level heading only_. 
Read this link https://html5accessibility.com/stuff/2021/07/08/preserved-in-html/
I think that will be better to use h2 for profile-name.
 ```<h1 class="profile-name">Tukach Kirill</h1>```
And ```<h1>My CV (or other heading)</h1>``` move up to top-level.